src/Simplex/Chat/ProfileGenerator.hs

```diff
@@ -0,0 +1,3163 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Simplex.Chat.ProfileGenerator where
+
+import Data.Text (Text)
+import Simplex.Chat.Types (Profile (..))
+import System.Random (randomRIO)
```

Comment:
we shouldn't use RIO, we have Random from cryptonite, let's not add libraries when not needed.

Reply:
ah, it's not RIO library - still, why not random we already use?